### PR TITLE
Insert important missing word, and polish wording throughout.

### PR DESCRIPTION
**Pull request by pat-rogers**

### CODE REVIEW

File: content/courses/ada-idioms/chapters/constructor_functions_for_abstract_data_types.rst [hunk 1]
```diff
@@ -82,15 +86,16 @@ example:
...
 
 In contrast, *concrete* types have an actual representation and can be used to
-declare objects. Likewise, concrete subprograms are callable, implemented
-units. In the following discussion, *abstract* has the OOP sense unless stated
+declare objects. Likewise, concrete subprograms are fully implemented, callable
+units. In the following discussion, *abstract* has this OOP sense unless stated
 otherwise.
 
-When a tagged type is extended, inherited constructor functions automatically
+We that definition in place, we can explain how Ada prevents constructor inheritance:
```
Comment (gusthoff) [inline content/courses/ada-idioms/chapters/constructor_functions_for_abstract_data_types.rst:93]: "We" => "With" (I guess...)

**Review by gusthoff — CHANGES_REQUESTED** [review]
Apart from a small typo, it looks good!

**Comment by pat-rogers** [comment]
On 26-Feb-24 4:42 PM, Gustavo A. Hoffmann wrote:
> ***@***.**** commented on this pull request.
> 
> ------------------------------------------------------------------------
> 
> In 
> content/courses/ada-idioms/chapters/constructor_functions_for_abstract_data_types.rst <https://github.com/AdaCore/learn/pull/1004#discussion_r1503404037>:
> 
>>  otherwise.
>   
> -When a tagged type is extended, inherited constructor functions automatically
> +We that definition in place, we can explain how Ada prevents constructor inheritance:
> 
> "We" => "With" (I guess...)

Yes indeed, good catch.

Can I ask you to make that change for me please?

File: content/courses/ada-idioms/chapters/constructor_functions_for_abstract_data_types.rst (hunk 1 above)
Comment (gusthoff) [reply content/courses/ada-idioms/chapters/constructor_functions_for_abstract_data_types.rst]: I've just fixed it.

**Review by gusthoff — APPROVED** [review]

**Comment by pat-rogers** [comment]
> I've just fixed it.

Thanks!

**Pull request merged**